pkg/test/client.go

```diff
@@ -31,6 +33,63 @@ func NewFakeClient(t T, initObjs ...client.Object) *FakeClient {
 	return &FakeClient{Client: cl, T: t}
 }
 
+func FakeSSA(fakeClient *FakeClient) {
+	fakeClient.MockPatch = FakeSSAPatchFunction(fakeClient)
```

Comment:
why not running this FakeSSAPatchFunction by default as part of the default FakeClient implementation? In the same way as it is currently done for the `Update` function.